cmake/linux/package_linux.sh.in

```diff
@@ -80,15 +80,19 @@ success "${APPDIR}"
 
 # Create a wrapper script which calls the lmms executable
 mv "$APPDIR/usr/bin/lmms" "$APPDIR/usr/bin/lmms.real"
-echo "#!/usr/bin/env bash
-DIR=\"\$( cd \"\$( dirname \"\${BASH_SOURCE[0]}\" )\" && pwd )\"
+# shellcheck disable=SC1083
```

Comment:
I just saw your comment, sorry. I don't have the time to look into it and don't see any easy answer.

If your problem is not described in the [wiki](https://github.com/koalaman/shellcheck/wiki/SC1083) as a known one. Maybe the best is to open an issue on the ShellCheck repo ?